app/controllers/candidate_interface/other_qualifications/details_controller.rb

```diff
@@ -7,13 +7,15 @@ def new
         current_application.application_qualifications.other.order(:created_at),
       )
       @wizard.save_state!
-      set_subject_data_for_autocomplete
+      set_subject_autocomplete_data
+      set_grade_autocomplete_data
```

Comment:
I think this data could live in the wizard or (more appropriately) a Details form object that we might extract. Because this code is currently in flux, I'm putting these methods in the controller for now. (fyi: @stevehook )